MultiVersions/Forge/main/beachparty/lang/zh_tw.json

```
@@ -0,0 +1,143 @@
+  "block.beachparty.palm_log": "棕櫚木原木",
+  "block.beachparty.palm_planks": "棕櫚木材",
+  "block.beachparty.palm_pressure_plate": "棕櫚木壓力板",
+  "block.beachparty.palm_sapling": "棕櫚木樹苗",
+  "block.beachparty.palm_sign": "棕櫚木告示牌",
+  "block.beachparty.palm_slab": "棕櫚木半磚",
+  "block.beachparty.palm_stairs": "棕櫚木階梯",
+  "block.beachparty.palm_tall_torch": "棕櫚木高火把",
+  "block.beachparty.palm_torch": "棕櫚木火把",
+  "block.beachparty.palm_torch_item": "棕櫚木火把",
+  "block.beachparty.palm_trapdoor": "棕櫚木地板門",
+  "block.beachparty.palm_wall_torch": "牆上的棕櫚木火把",
+  "block.beachparty.palm_wood": "棕櫚木塊",
+  "block.beachparty.pumpkin_cocktail": "南瓜雞尾酒",
+  "block.beachparty.radio": "收音機",
+  "block.beachparty.sand_bucket": "沙桶",
+  "block.beachparty.sand_bucket_block": "沙桶",
+  "block.beachparty.sand_dirty": "帶雜物的沙",
+  "block.beachparty.sand_pile": "沙堆",
+  "block.beachparty.sand_seastars": "帶海星的沙",
+  "block.beachparty.sand_slab": "沙半磚",
+  "block.beachparty.sandcastle": "沙堆城堡",
+  "block.beachparty.sandwaves": "海浪沙",
+  "block.beachparty.stripped_palm_log":"剝皮棕櫚木原木",
+  "block.beachparty.stripped_palm_wood":"剝皮棕櫚木塊",
+  "block.beachparty.sweetberries_cocktail": "草莓雞尾酒",
+  "block.beachparty.table": "棕櫚木桌",
+  "block.beachparty.tiki_bar":"熱帶風格吧檯",
+  "block.beachparty.tiki_chair":"熱帶風格木椅",
+  "entity.beachparty.pelican": "鵜鶘",
+  "entity.minecraft.villager.beachparty.barkeeper": "酒保",
+  "entity.minecraft.villager.beachparty.beach_boy": "沙灘服務生",
+  "gui.beachparty.recipebook.toggleRecipes.refridgable": "可冷藏",
+  "gui.beachparty.recipebook.toggleRecipes.shakeable": "可調製",
+  "item.beachparty.beach_hat": "遮陽帽",
+  "item.beachparty.bikini": "比基尼",
+  "item.beachparty.coconut_open": "剖開的椰子",
+  "item.beachparty.cooked_mussel_meat": "烤扇貝肉",
+  "item.beachparty.cooked_pelican": "烤鵜鶘肉",
+  "item.beachparty.crocs": "鱷魚鞋",
+  "item.beachparty.icecream_cactus": "仙人掌冰棒",
+  "item.beachparty.icecream_chocolate": "巧克力冰淇淋",
+  "item.beachparty.icecream_coconut": "椰子冰棒",
+  "item.beachparty.icecream_melon": "西瓜冰棒",
+  "item.beachparty.icecream_sweetberries": "草莓冰淇淋",
+  "item.beachparty.overgrown_disc": "Overgrown 唱片",
+  "item.beachparty.overgrown_disc.desc": "MissLilitu - Over the Rainbow",
+  "item.beachparty.palm_boat": "棕櫚木船",
+  "item.beachparty.palm_chest_boat": "儲物箱棕櫚木船",
+  "item.beachparty.pelican_spawn_egg": "鵜鶘 生怪蛋",
+  "item.beachparty.pool_noodle": "浮力游泳棒",
+  "item.beachparty.raw_mussel_meat": "生扇貝肉",
+  "item.beachparty.raw_pelican": "生鵜鶘肉",
+  "item.beachparty.rubber_ring_axolotl": "六角恐龍泳圈",
+  "item.beachparty.rubber_ring_blue": "藍色泳圈",
+  "item.beachparty.rubber_ring_pelican": "鵜鶘泳圈",
+  "item.beachparty.rubber_ring_pink": "粉紅色泳圈",
+  "item.beachparty.rubber_ring_stripped": "條紋泳圈",
+  "item.beachparty.seashell": "扇貝",
+  "item.beachparty.sunglasses": "太陽眼鏡",
+  "item.beachparty.swim_wings": "泳翼",
+  "item.beachparty.trunks": "泳褲",
+  "creativetab.beachparty.tab": "共度時光｜沙灘派對",
+  "subtitles.beachparty.sound.ambient_beach": "海浪聲",
```

根據遊戲內實測，前面的 `subtitles.` 都要刪掉，不然字幕會顯示翻譯鍵。
原文也有同樣的問題。
所有字幕都是。
```suggestion
  "beachparty.sound.ambient_beach": "海浪聲",
```